openapi/openapi/src/main/scala/endpoints4s/openapi/JsonSchemas.scala

```diff
@@ -127,12 +134,83 @@ trait JsonSchemas extends algebra.JsonSchemas with TuplesSchemas {
 
     }
 
-    case class Field(
-        name: String,
-        tpe: DocumentedJsonSchema,
-        isOptional: Boolean,
-        documentation: Option[String]
-    )
+    class Field(
+        val name: String,
+        val tpe: DocumentedJsonSchema,
+        val isOptional: Boolean,
+        val defaultValue: Option[ujson.Value],
+        val documentation: Option[String]
+    ) extends Serializable
+        with Product
+        with Equals {
+
+      override def canEqual(other: Any): Boolean = other.isInstanceOf[Field]
+      override def productArity: Int = 5
+      override def productPrefix: String = "Field"
+      override def productElement(idx: Int): Any = idx match {
+        case 0 => name
+        case 1 => tpe
+        case 2 => isOptional
+        case 3 => defaultValue
+        case 4 => documentation
+        case _ => throw new IndexOutOfBoundsException(idx.toString)
+      }
+      override def productElementNames: Iterator[String] =
+        Iterator("name", "tpe", "isOptional", "defaultValue", "documentation")
+      override def toString: String = s"Field($name,$tpe,$isOptional,$defaultValue,$documentation)"
+      override def hashCode: Int = Hashing.hash(name, tpe, isOptional, defaultValue, documentation)
+      override def equals(other: Any): Boolean = other match {
+        case field: Field =>
+          name == field.name && tpe == field.tpe && isOptional == field.isOptional &&
+            defaultValue == field.defaultValue && documentation == field.documentation
+        case _ => false
+      }
+
+      def this(
+          name: String,
+          tpe: DocumentedJsonSchema,
+          isOptional: Boolean,
+          documentation: Option[String]
+      ) = this(name, tpe, isOptional, None, documentation)
+
+      def copy(
```

Comment:
This method should be deprecated, instead we should add methods like `withName`, `withType`, etc.